internal/xds/translator/route.go

```diff
@@ -239,9 +241,27 @@ func buildXdsWeightedRouteAction(httpRoute *ir.HTTPRoute) *routev3.RouteAction {
 				Clusters: clusters,
 			},
 		},
+		IdleTimeout: idleTimeout(httpRoute),
 	}
 }
 
+func idleTimeout(httpRoute *ir.HTTPRoute) *durationpb.Duration {
```

Comment:
I'd vote for below logic until we expose this as an API field
```
idleTimeout = 1hr
if idleTimout < requestTimeout {
    idleTimout = requestTimeout
}

Reply:
I'm ok with this, with the caveat that it should still be set to zero so the request timeout will function per GEP-1742:

> A zero-valued timeout ("0s") MUST be interpreted as disabling the timeout
ref: https://gateway-api.sigs.k8s.io/geps/gep-1742/#timeout-values

This should work per envoy docs:

> value of 0 will completely disable the route’s idle timeout, even if a connection manager stream idle timeout is configured

ref: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-field-config-route-v3-routeaction-idle-timeout

Reply:
yah agree with that